jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -96,7 +96,8 @@ def _to_np_dtype(dtype) -> NpDType:
   else:
     np_dtype = None
 
-  if prim in [lax.min_p, lax.max_p]:
+  if prim in [lax.min_p, lax.max_p, lax.reduce_window_min_p,
```

Comment:
Wouldn't this lead to changes in the unimplemented primitives documentation? Maybe you need to re-generate that documentation?